server/__init__.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from flask import Flask, request, send_file, session, jsonify
+from server.settings import *
+
+
+def create_app(test_config=None):
+    app = Flask(__name__)
+
+    validate_server_settings()
+
+    if test_config is not None:
+        app.config.from_mapping(test_config)
+    
+    from .home import views as home_views
+    app.register_blueprint(home_views.bp)
+    from .jobs import interfaces as job_interfaces
+    app.register_blueprint(job_interfaces.bp)
+    from .jobs import actions as job_actions
+    app.register_blueprint(job_actions.bp)
```

Comment:
`jobs/interfaces.py` and `jobs/actions.py` serve the APIs interacting with the job dispatcher and tracker.